draft-ietf-tls-dtls-connection-id.md

```diff
@@ -534,11 +534,11 @@ The topic of peer address updates is discussed in {{peer-address-update}}.
 
 #  IANA Considerations
 
-This document requests three actions by IANA. 
+This document requests three actions by IANA.
 
 ## Extra Column to TLS ExtensionType Values Registry
 
-IANA is requested to add an extra column to the
+IANA is requested to add an extra column named "DTLS-Only" to the
 "TLS ExtensionType Values" registry to indicate whether an extension is only
 applicable to DTLS and to include this document as an additional reference
 for the registry.
```

Comment:
IIRC in the thread with IANA we did state the expectations for how this column is populated (Y and N), so I think it's okay to not say more here.